operations/service_deploy_smoke_utils.go

```diff
@@ -99,22 +99,34 @@ func checkTaskByCommit(username, key string) error {
 
 	var builds []apimodels.APIBuild
 	var build apimodels.APIBuild
+
+	time.Sleep(10 * time.Second)
+	// trigger repotracker
+	for i := 0; i < 5; i++ {
```

Comment:
I'm not convinced this is necessary: adding the time.Sleep(10 * time.Second) made this work pretty much every time for me, but possibly to avoid it being flaky it's still good to have a loop and a sleep.